portal-api/src/scheduler/jobs/helpers/utilisation-report-helpers.js

```diff
@@ -0,0 +1,47 @@
+const { startOfMonth, format, subMonths } = require('date-fns');
+const externalApi = require('../../../external-api/api');
+const { addBusinessDaysWithHolidays } = require('../../../utils/date');
+
+/**
+ * Returns the utilisation report due date for the current month
+ * @returns {Promise<Date>}
+ */
+const getReportDueDate = async () => {
+  const bankHolidays = await externalApi.bankHolidays.getBankHolidayDatesForRegion('england-and-wales');
+  return addBusinessDaysWithHolidays(startOfMonth(new Date()), process.env.PORTAL_REPORTING_DUE_DATE_BUSINESS_DAYS_FROM_START_OF_MONTH, bankHolidays);
+};
+
+/**
+ * Returns the utilisation report due date for the current month in 'do MMMM yyyy' format
+ * @returns {Promise<string>}
+ */
+const getFormattedReportDueDate = async () => {
+  const reportDueDate = await getReportDueDate();
+  return format(reportDueDate, 'do MMMM yyyy');
```

Comment:
I don't know what the correct approach should be here, but the designs I've interacted with use `'d MMMM yyyy'` - maybe worth asking?

Reply:
Thanks, will update to match elsewhere 👍